app/main.py

```diff
@@ -1,13 +1,31 @@
 def format_linter_error(error: dict) -> dict:
-    # white your code here
-    pass
+    return dict(line=error["line_number"],
+                column=error["column_number"],
+                message=error["text"],
+                name=error["code"],
+                source="flake8")
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # white your code here
-    pass
+    return {"errors": [
+        dict(line=error["line_number"],
+             column=error["column_number"],
+             message=error["text"],
+             name=error["code"],
+             source="flake8")
```

Comment:
You have implemented "format_linter_error" above, why did you duplicate code from that function?